test/strategy/liquity/LiquityStrategy.fork.spec.ts

```diff
@@ -365,7 +366,7 @@ describe('Liquity Strategy (mainnet fork tests)', () => {
         EXPECTED_ETH_REWARD,
       );
 
-      expect(await strategy.investedAssets()).to.eq('4998816139652613137823');
+      expect(await strategy.investedAssets()).to.eq('5000736699812217881367');
```

Comment:
I would suggest adding another test for this since you introduced an `if` branch in the code

Reply:
I dont understand. What type of test do you mean?

Reply:
I meant adding another test that is covering the code path when `ethBalance == 0` condition is met